packages/api/migrations/20200820002013-create-nomination.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Nominations', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      status: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      dateReceived: {
+        type: Sequelize.DATE
+      },
+      providerFirstName: {
+        type: Sequelize.STRING
+      },
+      providerLastName: {
+        type: Sequelize.STRING
+      },
+      providerNumber: {
+        type: Sequelize.INTEGER
+      },
+      emailAddress: {
+        type: Sequelize.STRING
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      validated: {
```

Comment:
What kind of validation does this column name refer to?

Reply:
That column name refers to whether the nomination itself has been validated or not by an admin, I believe. Will double check with Travell.

Reply:
Just checked with Travell. It's for email validation.